packages/interbit/src/scripts/build.js

```diff
@@ -3,40 +3,30 @@ const promisify = require('util').promisify
 const exec = promisify(require('child_process').exec)
 const path = require('path')
 
-const startInterbit = require('../chainManagement/startInterbit')
-const getArtifactsLocation = require('../args/getArtifactsLocation')
-const getManifest = require('../args/getManifest')
-const getConfig = require('../args/getConfig')
-const { generateManifest } = require('./generateManifest')
-const { updateIndexHtmls } = require('../chainManagement/updateIndexHtml')
+const { startInterbit } = require('../chainManagement')
+const { generateManifest } = require('../manifest')
+const { updateIndexHtmls, writeJsonFile } = require('../file')
 
-const build = async () => {
-  const interbitConfig = getConfig()
-  const interbitManifest = getManifest()
+const build = async options => {
+  const { config, manifest, artifacts } = options
 
-  const artifactsLocation = getArtifactsLocation()
-  const location = path.relative(process.cwd(), artifactsLocation)
-
-  // TODO: Copy the specified app builds into dist/builds/dirname #9
+  const location = path.relative(process.cwd(), artifacts)
 
   setupDist(location)
 
-  const covenantFilenames = await packCovenants(
-    location,
-    interbitConfig.covenants
-  )
+  const covenantFilenames = await packCovenants(location, config.covenants)
 
   const newManifest = generateManifest(
     location,
-    interbitConfig,
+    config,
     covenantFilenames,
-    interbitManifest
+    manifest
   )
 
-  writeManifestToFile(location, newManifest)
+  writeJsonFile(`${location}/interbit.manifest.json`, newManifest)
 
```

Comment:
--manifest is the manifest being read into the build for comparison. 

Part of another ticket & cascading deployment is to output a name corresponding to the input config file.

So if you had interbit.prod.config.js it would output interbit.prod.manifest.json.

TBD